### ISSUE TITLE
Make a release

### ISSUE THREAD

**Issue by cwmeijer** [dianna-ai/dianna#281]
blocked by #254

**Comment by elboyran** [comment]
[JOSS paper release](https://github.com/dianna-ai/dianna/releases/tag/v0.4.0)

**Comment by elboyran** [comment]
The PyPi for the build is failing.

**Comment by cwmeijer** [comment]
I saw the version wasn't bumped to 0.4.0, which means that version 0.3.0 was being pushed to pypi again, which doesn't allow pushing the same version twice. This caused the failure of the workflow.
In an attempt to fix it, yesterday, I deleted Elena's 0.4.0 release, bumped the version, and made another release. I made the mistake of using the same tag though, 0.4.0, so my version bumping wasn't included as that came right after that tag. I'm now making a release 0.4.1 and everything should be fine.
